src/components/Header.js

```diff
@@ -0,0 +1,46 @@
+import { useSelector } from 'react-redux';
+import { NavLink } from 'react-router-dom';
+import '../scss/style.scss';
+import { FaAngleLeft, FaMicrophone, FaCog } from 'react-icons/fa';
+
+const Header = () => {
+  const home = useSelector((state) => state.executive.home);
+
+  return (
+    <div className="NavBar-Container">
+      <Heading home={home} />
+    </div>
+  );
+};
+
+/* eslint-disable react/prop-types */
```

Comment:
- [ ] Remove the eslint disable rule here and include prop-types in the component.